src/commands/private.ts

```diff
@@ -0,0 +1,132 @@
+import { logger } from '../utilities/logger';
+import { CommandDef } from './command-def';
+import { MessageEmbed, TextChannel } from 'discord.js';
+
+export const privateCmd: CommandDef = {
+  prefix: 'private',
+  description: 'Creates a private channel for moderators and the target user.',
+  usage: 'private <user>',
+  command: async (message, { config }) => {
+    try {
+      //check for appropriate permissions
+      if (!message.member?.hasPermission('KICK_MEMBERS')) {
+        logger.warn(
+          `${message.author.username} did not have the correct permissions.`
+        );
+        message.channel.send(
+          'Sorry, this command is restricted to moderators.'
+        );
+        return;
+      }
```

Comment:
Is this the right permission check? Maybe we can use the `MOD_ROLE` environment variable check instead?

Reply:
That's a good approach, actually.